src-tauri/tauri.conf.json

```diff
@@ -33,7 +33,7 @@
       "macOS": {
         "frameworks": [],
         "minimumSystemVersion": "",
-        "exceptionDomain": "",
+        "exceptionDomain": "http://localhost:3009",
```

Comment:
Is this for dev?

Reply:
yes, sometimes it complains about security policy, [docs](https://tauri.app/v1/guides/building/macos/#application-bundle-customization) recommend adding `localhost`